### DISCUSSION TITLE
Support DASH Video

### DISCUSSION THREAD

**Discussion by NoLiners** [vidstack/player#915] [Q&A]
When Vidstack will support DASH video, I just know that VimeJs has supported it with great success, while for Plyr it doesn't support the latest DASH.

**Comment by sturlath** [comment]
I would be interested to know this also

**Comment by mihar-22** [comment]
The straight forward answer is my plate is full and there's higher priority things so not for a while. Once the code base stabilizes around 1.0 I'm more than happy to accept/guide a PR.

**Reply by Dmitry-DM** [reply]
Hello @mihar-22. It's been a while since the 1.0 release. May I know what the plans are for DASH?

**Reply by mihar-22** [reply]
This is the 1.0 pre-release, not main.

**Reply by Dmitry-DM** [reply]
@mihar-22 any approximate dates for when the main version 1.0 will be released?
